frontend/src/Settings/UI/UISettings.js

```diff
@@ -220,6 +220,13 @@ class UISettings extends Component {
                       helpTextWarning={translate('BrowserReloadRequired')}
                       onChange={onInputChange}
                       {...settings.uiLanguage}
+                      {...settings.errors}
+                      errors={
+                        languages.some((language) => language.key === settings.uiLanguage.value) ?
+                          [] :
+                          [
+                            { message: translate('InvalidUILanguage') }
+                          ]}
```

Comment:
It wasn't super clear what I meant with my previous comment, but it'll need to be like this to spread the server errors correctly.

```suggestion
                      errors={
                        languages.some((language) => language.key === settings.uiLanguage.value) ?
                          settings.error :
                          [
                            ...settings.error,
                            { message: translate('InvalidUILanguage') }
                          ]}
```

Reply:
this works when the language is valid, but throws an error when the language in the db is invalid:
```
UISettings.js:227 Uncaught TypeError: settings.error is not iterable
    at UISettings.render (UISettings.js:227:41)```

doing this works but not sure if thats right?

```
                      errors={
                        languages.some((language) => language.key === settings.uiLanguage.value) ?
                          settings.error :
                          [
                            { ...settings.error },
                            { message: translate('InvalidUILanguage') }
                          ]}
                          ```

Reply:
Oops, should be `errors` not `error`

Reply:
same issue though
```
UISettings.js:227 Uncaught TypeError: settings.errors is not iterable
    at UISettings.render (UISettings.js:227:41)
    ```

Reply:
Double oops, it's `settings.uiLanguage.errors`

Reply:
> Double oops, it's `settings.uiLanguage.errors`... not sure why I made up `settings.errors`

thanks, that was it! still learning my around js, react, and C#, and trying to find ways to use JetBrains Rider to investigate variables. Lot of learning :)